db/data/20220930022007_populate_room_limit_role_permission.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class PopulateRoomLimitRolePermission < ActiveRecord::Migration[7.0]
+  def up
+    Permission.create! [
```

Comment:
I think you can add this directly to the `PopulatePermissions`.

Reply:
I was told by @farhatahmad to create new migration file for these since we released alpha and don't want people to have to nuke their db for the new changes

Reply:
Yeah, good point!